src/app/frontend/replicasetlist/replicasetlist_stateconfig.js

```diff
@@ -24,16 +25,28 @@ import ReplicaSetListController from './replicasetlist_controller';
  * @ngInject
  */
 export default function stateConfig($stateProvider) {
-  $stateProvider.state(replicasets, {
-    controller: ReplicaSetListController,
-    controllerAs: 'ctrl',
-    url: replicasetsUrl,
-    resolve: {
-      'replicaSets': resolveReplicaSets,
-    },
-    templateUrl: 'replicasetlist/replicasetlist.html',
-    onEnter: redirectIfNeeded,
-  });
+  $stateProvider.state(
+                    replicasets,
+                    {
+                      controller: ReplicaSetListController,
+                      controllerAs: 'ctrl',
+                      url: replicasetsUrl,
+                      resolve: {
+                        'replicaSets': resolveReplicaSets,
+                      },
+                      templateUrl: 'replicasetlist/replicasetlist.html',
+                      onEnter: redirectIfNeeded,
+                    })
+      .state(zerostate, {
+        views: {
+          '@': {
+            controller: ZeroStateController,
+            controllerAs: 'ctrl',
+            templateUrl: 'replicasetlist/zerostate/zerostate.html',
+          },
+        },
+        url: '/zerostate',
+      });
```

Comment:
Well, at least it is machine formatted and we cannot argue over this :)

It looks a lot better when split into two states:

```js
$stateProvider.state(replicasets, {
    controller: ReplicaSetListController,
    controllerAs: 'ctrl',
    url: replicasetsUrl,
    resolve: {
      'replicaSets': resolveReplicaSets,
    },
    templateUrl: 'replicasetlist/replicasetlist.html',
    onEnter: redirectIfNeeded,
  });
  $stateProvider.state(zerostate, {
    views: {
      '@': {
        controller: ZeroStateController,
        controllerAs: 'ctrl',
        templateUrl: 'replicasetlist/zerostate/zerostate.html',
      },
    },
    url: '/zerostate',
  });
```